h264/parse.go

```diff
@@ -20,11 +20,16 @@ import (
 	"github.com/pkg/errors"
 )
 
-type macroblockPredictionMode uint8
+type mbPartPredMode int8
```

Comment:
Is there a reason for reducing the length of this name?

Reply:
I'm trying to keep the naming more consistent with the specs.

Reply:
Clarity is more important than matching the spec; if matching the spec helps, do it, otherwise improve on it. A good example is the crypto/tls package and its constants/field names compared with the relevant RFCs.

Reply:
I really think matching the specs as much as possible (unless it breaks go style) is good. It helps to find things in the spec more easily.

Reply:
I disagree.

Reply:
Okay, i'll move away from that idea. Please take a look at the code and create issues if you see naming that could be better.